src/aind_slims_api/models/attachment.py

```diff
@@ -0,0 +1,34 @@
+"""Model for a record in the Attachment table in SLIMS."""
+
+from pydantic import Field
+
+from aind_slims_api.models.base import SlimsBaseModel
+
+
+class SlimsAttachment(SlimsBaseModel):
+    """Model for a record in the Attachment table in SLIMS.
+
+    Examples
+    --------
+    >>> from aind_slims_api import SlimsClient
+    >>> client = SlimsClient()
+    >>> rig_metadata_attachment = client.fetch_model(
+    ...  SlimsAttachment,
+    ...  name="rig323_EPHYS1_OPTO_2024-02-12.json"
+    ... )
+    >>> rig_metadata = client.fetch_attachment_content(
+    ...  rig_metadata_attachment
+    ... ).json()
+    >>> rig_metadata["rig_id"]
+    '323_EPHYS1_OPTO_2024-02-12'
+    """
+
+    pk: int = Field(..., alias="attm_pk")
+    name: str = Field(..., alias="attm_name")
+    _slims_table = "Attachment"
+
+
+if __name__ == "__main__":
```

Comment:
I run into exceptions running this locally. I think the doc tests are doing io stuff? We should probably have the doc tests not require a connection to the db.

Reply:
Yeah, the doctests are included as integration tests and are intended to be manually triggered during development. This seems like a good idea in my mind because this is a client and the documentation should always be accurate.

Reply:
I agree, but integration tests are usually added under the testing package. They also usually run against a mock server. It's fine to have examples in the doc strings, but we don't want testing code inside of src. Since this is mostly a wrapper around https://github.com/genohm/slims-python-api, we can probably use that library as an example. It's also something that we can add as a separate PR.

Reply:
resolved in [d5e6d27](https://github.com/AllenNeuralDynamics/aind-slims-api/pull/20/commits/d5e6d27f39eda480803b55daf2f30d461008b877)